### PR TITLE
Enhancement: Enable `trailing_comma_in_multiline` fixer

### PR DESCRIPTION
**Pull request by localheinz**
This pull request

- [x] enables the `trailing_comma_in_multiline` fixer
- [x] runs `make coding-standards`

Follows #559.

💁‍♂️ For reference, see https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v3.40.2/doc/rules/control_structure/trailing_comma_in_multiline.rst.

### CODE REVIEW

File: cal.php [hunk 1]
```diff
@@ -142,7 +142,7 @@
                    urlencode($month),
                    urlencode($year),
                    htmlentities($monthName),
-                   htmlentities($year));
+                   htmlentities($year), );
```
[the remark this answers is not in the document]
Comment (cmb69) [reply cal.php:145]: Indeed, there is `method_argument_space`.

**Comment by derickr** [comment]
This has many conflicts.

**Comment by localheinz** [comment]
Rebased!

File: git-php.php [hunk 2]
```diff
@@ -363,7 +363,7 @@ class="max" value="<?php if (isset($_POST['email'])) echo clean($_POST['email'])
 $purposes = ["Learning PHP", "Coding in PHP", "Reading the PHP source",
     "Using PHP extensions", "Creating experimental PHP extensions",
     "Submitting a patch to PHP", "Adding notes to the documentation",
-    "Writing web pages with PHP"];
+    "Writing web pages with PHP", ];
```
Comment (Girgias) [inline git-php.php:366]: @localheinz is there a fixer or a PR already for:
```suggestion
    "Writing web pages with PHP",
];
```
?

**Review by Girgias — COMMENTED** [review]
Mostly LGTM, just a couple of questions

**Comment by Girgias** [comment]
Needs a rebase again

**Comment by localheinz** [comment]
Rebased, @Girgias!

File: .php-cs-fixer.php [hunk 3]
```diff
@@ -51,6 +51,12 @@
         ],
         'strict_param' => true,
         'switch_case_space' => true,
+        'trailing_comma_in_multiline' => [
+            'elements' => [
+                'arguments',
+                'arrays',
```
[the remark this answers is not in the document]
Comment (Girgias) [reply .php-cs-fixer.php:57]: I think this probably makes sense as a follow-up PR to not clog this up any longer

**Comment by localheinz** [comment]
Thank you, @cmb69, @derickr, @Girgias, and @saundefined!

**Pull request merged**